README.md

```diff
@@ -0,0 +1,24 @@
+# pidpath
+
+Retrieve the absolute path to the executable of a running process.
+
+## Usage
+```
+pidpath <pid>
+```
+
+## Example
+
+```sh
+ps aux | grep bash
+vagrant   3249  0.0  0.2  23156  5220 pts/0    Ss   22:57   0:00 -bash
+
+pidpath 3249
+/bin/bash
+```
+
+## Install
+
+```sh
+go get github.com/cirocosta/pidpath
```

Comment:
I think this would need to be adjusted actually now that the `main` package is under `cmd` 🤔  when having a main package under `cmd` I usually create an extra one with the name of the binary I want to produce (like, `cmd/pidpath` where I then include the `main` package), wdyt of that pattern?

Reply:
> I think this would need to be adjusted actually now that the main package is under cmd 🤔

ah, I suppose that depends on _what_ you'd be trying to install. I believe `go get […]` is typically used for adding a given package as a dependency to the current project. for this, `github.com/cirocosta/pidpath` would be the correct identifier post-PR.

I suppose I did not consider the aspect of installing the binary directly though. for that, the path would indeed need to be adjusted. consulting `go help`, I believe the right command for this would also be `go install […]`

> wdyt of that pattern?

so far I've followed that pattern too. here though I figured I'd simplify this here since there is not gonna be a second CLI for this small package. thinking this could just be `/cmd`. however, trying `go install` with this setup now, I can see go ends up installing a `cmd` binary 🙈 I now see why the `cmd/{name}` pattern is used even in projects with only ever a single CLI.

should be easy to address; will push a fix shortly!